ts/features/itwallet/presentation/screens/ItwPresentationEidVerificationExpiredScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import { useMemo } from "react";
+import { BodyProps } from "@pagopa/io-app-design-system";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { ITW_ROUTES } from "../../navigation/routes";
+
+export const ItwPresentationEidVerificationExpiredScreen = () => {
+  const navigation = useIONavigation();
+
+  const startEidReissuing = () => {
+    navigation.navigate(ITW_ROUTES.MAIN, {
+      screen: ITW_ROUTES.IDENTIFICATION.MODE_SELECTION,
+      params: {
+        eidReissuing: true
+      }
+    });
+  };
+
+  const bodyPropsArray: Array<BodyProps> = useMemo(
```

Comment:
This `useMemo` is unnecessary, imho. It does not contain any expensive calculation, it does not depend on anything and the parent component does not seem to have any re-rendering. 
I think we can safely remove it. What do you think?